app/src/main/java/com/epfl/beatlink/viewmodel/profile/ProfileViewModel.kt

```diff
@@ -89,7 +110,12 @@ open class ProfileViewModel(
     _isProfileUpdated.value = false
   }
 
-  fun getUsername(userId: String, onResult: (String?) -> Unit) {
+  fun clearSelectedUser() {
```

Comment:
Here as well, don't forget to give documentation for new functions (there are multiple in this file)